data/views/redfish-1.0/redfish.1.0.0.computersystem.1.0.0.json

```diff
@@ -60,9 +59,6 @@
     "SimpleStorage": {
         "@odata.id": "<%= basepath %>/Systems/<%= identifier %>/SimpleStorage"
     },
-    "Storage": {
```

Comment:
Curious as to why this is being removed when it was added ~1 month ago in this pr: https://github.com/RackHD/on-http/pull/708/files.  

original tracking story: https://rackhd.atlassian.net/browse/RAC-5361

Reply:
Storage should only exist in redfish computer systems that are version 1.1.1 or greater.  1.0.0 should not have that field.  The validation tests pick this up as an error.  Also, this is a WIP so I'm guessing its not ready for prime time.

Reply:
gotcha and thanks for clarification!  by chance, is there a story tracking the update to the 1.1.1 version?

Reply:
No story specifically to transitioning ComputerSystem to 1.1.1.  The fields that are added for 1.1.1 are not required fields so there wouldn't be 1.1.1 conformance issues due to lack of support.